packages/block-editor/src/store/reducer.js

```diff
@@ -1887,3 +1881,34 @@ export default combineReducers( {
 	temporarilyEditingAsBlocks,
 	blockVisibility,
 } );
+
+function withAutomaticChangeReset( reducer ) {
+	return ( state, action ) => {
+		const nextState = reducer( state, action );
+
+		if ( ! state ) {
+			return nextState;
+		}
+
+		if (
+			nextState.blocks === state.blocks &&
+			nextState.selection === state.selection
```

Comment:
So the question is: Is there potentially an action that updates either selection or blocks but shouldn't update "automaticChangeStatus"?

Reply:
I don't think so

Reply:
So far every time we had this issue it's because of some other non content state